Rakefile

```diff
@@ -1,9 +1,15 @@
 task :dev do
-  sh "bundle exec rackup --host 0.0.0.0 --port 8080 --env development"
+  unless system("which foreman", out: File::NULL)
+    raise "`foreman` is now required for process management. `sudo gem install foreman`"
```

Comment:
Any reason why we recommend `sudo gem install` over `bundle install`?

Reply:
`foreman` doesn't (and shouldn't according to the foreman Readme, and https://github.com/ddollar/foreman/pull/678#issuecomment-398211757) live in the Gemfile.

Reply:
That makes sense, let's add a comment here with a reference to the GH issue